examples/superminimal/input.py

```diff
@@ -0,0 +1,55 @@
+# Data sources
```

Comment:
Please add a header comment, something like "# taken from the RMG-Py superminimal example ..." and give the URL